src/manifest/api/Scopes.ts

```diff
@@ -8,180 +8,424 @@
-  CALLS_WRITE = 'calls.write',
-  CALLS_DELETE = 'calls.delete',
+  COMPLIANCE_REQUESTS_ALL = 'complianceRequests.all',
+  COMPLIANCE_REQUESTS_DELETE = 'complianceRequests.delete',
+  COMPLIANCE_REQUESTS_READ = 'complianceRequests.read',
+  COMPLIANCE_REQUESTS_WRITE = 'complianceRequests.write',
 
-  CUSTOM_DUTIES_ALL = 'customDuties.all',
-  CUSTOM_DUTIES_READ = 'customDuties.read',
-  CUSTOM_DUTIES_WRITE = 'customDuties.write',
-  CUSTOM_DUTIES_DELETE = 'customDuties.delete',
+  CONTENT_CATEGORY_OWNERSHIPS_ALL = 'contentCategoryOwnerships.all',
+  CONTENT_CATEGORY_OWNERSHIPS_DELETE = 'contentCategoryOwnerships.delete',
+  CONTENT_CATEGORY_OWNERSHIPS_READ = 'contentCategoryOwnerships.read',
+  CONTENT_CATEGORY_OWNERSHIPS_WRITE = 'contentCategoryOwnerships.write',
 
   CONTENT_CATEGORIES_ALL = 'contentCategories.all',
+  CONTENT_CATEGORIES_DELETE = 'contentCategories.delete',
   CONTENT_CATEGORIES_READ = 'contentCategories.read',
   CONTENT_CATEGORIES_WRITE = 'contentCategories.write',
-  CONTENT_CATEGORIES_DELETE = 'contentCategories.delete',
 
   CONTENT_CATEGORY_MEMBERSHIPS_ALL = 'contentCategoryMemberships.all',
+  CONTENT_CATEGORY_MEMBERSHIPS_DELETE = 'contentCategoryMemberships.delete',
   CONTENT_CATEGORY_MEMBERSHIPS_READ = 'contentCategoryMemberships.read',
   CONTENT_CATEGORY_MEMBERSHIPS_WRITE = 'contentCategoryMemberships.write',
-  CONTENT_CATEGORY_MEMBERSHIPS_DELETE = 'contentCategoryMemberships.delete',
+
+  CREATE_ACCOUNTS = 'create_accounts',
+  CREATE_CALLS = 'create_calls',
+  CREATE_PROSPECTS = 'create_prospects',
+
+  CUSTOM_VALIDATIONS_ALL = 'customValidations.all',
+  CUSTOM_VALIDATIONS_DELETE = 'customValidations.delete',
+  CUSTOM_VALIDATIONS_READ = 'customValidations.read',
+  CUSTOM_VALIDATIONS_WRITE = 'customValidations.write',
+
+  CUSTOM_DUTIES_ALL = 'customDuties.all',
+  CUSTOM_DUTIES_DELETE = 'customDuties.delete',
+  CUSTOM_DUTIES_READ = 'customDuties.read',
+  CUSTOM_DUTIES_WRITE = 'customDuties.write',
+
+  DEAL_RISK_FACTORS_ALL = 'dealRiskFactors.all',
+  DEAL_RISK_FACTORS_DELETE = 'dealRiskFactors.delete',
+  DEAL_RISK_FACTORS_READ = 'dealRiskFactors.read',
+  DEAL_RISK_FACTORS_WRITE = 'dealRiskFactors.write',
 
   DUTIES_ALL = 'duties.all',
+  DUTIES_DELETE = 'duties.delete',
   DUTIES_READ = 'duties.read',
   DUTIES_WRITE = 'duties.write',
-  DUTIES_DELETE = 'duties.delete',
 
   EMAIL_ADDRESSES_ALL = 'emailAddresses.all',
+  EMAIL_ADDRESSES_DELETE = 'emailAddresses.delete',
   EMAIL_ADDRESSES_READ = 'emailAddresses.read',
   EMAIL_ADDRESSES_WRITE = 'emailAddresses.write',
-  EMAIL_ADDRESSES_DELETE = 'emailAddresses.delete',
 
   EVENTS_ALL = 'events.all',
+  EVENTS_DELETE = 'events.delete',
   EVENTS_READ = 'events.read',
   EVENTS_WRITE = 'events.write',
-  EVENTS_DELETE = 'events.delete',
+
+  EXPERIMENTS_ALL = 'experiments.all',
+  EXPERIMENTS_DELETE = 'experiments.delete',
+  EXPERIMENTS_READ = 'experiments.read',
+  EXPERIMENTS_WRITE = 'experiments.write',
+
+  FAMILY_NAME = 'family_name',
```

Comment:
Is it correct that we don't use cammel case here (as everywhere else)?

Reply:
Yes. Those values are taken from production apps